libafl_frida/build.rs

```diff
@@ -7,8 +7,10 @@ fn main() {
     }
 
     // Force linking against libc++
-    #[cfg(unix)]
-    println!("cargo:rustc-link-lib=dylib=c++");
+    cc::Build::new()
+        .cpp(true)
+        .file("src/stub.cpp")
+        .compile("libcppstub.a");
```

Comment:
I made this change because I could not build this example locally without the change. This is the recommended way to do this, as we've been told in previous PRs (cannot find an exact reference rn); C++ linking in the previous way was not portable.

Reply:
you need `libc++1-18 libc++-18-dev libc++abi1-18 libc++abi-18-dev` (from llvm)
I think the previous line is necessary as the ASAN symbols comes from llvm libc++

Reply:
You can definitely install clang w/ ASAN without libc++ (and libc++ is just often not necessary). Besides, we should be providing examples with good practices.

Reply:
nono it's not about installing or using asan from clang. 

In frida asan, we looks for certain symbols in the target program, and if it matches with the pre-computed function name/signatures list.
And we have hardcoded ".so" name to tell from which shared library you match the name
https://github.com/AFLplusplus/LibAFL/blob/main/libafl_frida/src/asan/asan_rt.rs#L615

libstdc++.so is not listed here so it has to use libc++